src/frameworks/drupal8/drush.md

```diff
@@ -10,10 +10,14 @@ commands at [Drush.org](http://www.drush.org). If you have never used
 Drush before, you can learn more about it on the [Drush GitHub
 Repository](https://github.com/drush-ops/drush#description)
 
-Platform.sh has Drush installed by default, so all environments can utilize Drush commands in the development process. You can use the CLI to set up Drush aliases, to easily run Drush commands on
-specific remote Platform.sh environments.
+- Platform.sh Drupal templates have Drush installed automatically.
+
+- All custom environments can utilize Drush commands in the development process if they have `drush` configured as a composer requirement and added it to the environment's `$PATH`.[¹](#install-drush-in-custom-projects)
```

Comment:
Internal linking with the doc: Great.  But we don't use footnote-ish style anywhere else in the docs.  They're bad for accessibility.  Instead, make a part of the previous sentence the link or add a sentence like "See the section below on 
 (link) installing Drush in custom projects (/link)"